kubecost-exporter.md

```diff
@@ -46,17 +46,18 @@ Done! Kubecost is now exporting cost metrics. See the following sections for dif
 | node_gpu_hourly_cost | Hourly cost per GPU on this node  |
 | node_ram_hourly_cost   | Hourly cost per Gb of memory on this node                       |
 | node_total_hourly_cost   | Total node cost per hour                       |
+| kubecost_load_balancer_cost   | Hourly cost of a load balancer                 |
```

Comment:
Add these to PROMETHEUS.md too?

Reply:
100% I'll do in a separate PR if that works for you.

Reply:
https://github.com/kubecost/cost-model/pull/709